src/Core/src/Handlers/ContentView/ContentViewHandler.Windows.cs

```diff
@@ -55,5 +57,16 @@ protected override void DisconnectHandler(ContentPanel platformView)
 
 			base.DisconnectHandler(platformView);
 		}
+
+		static UI.Xaml.FrameworkElement? CreateContent(IView content, IMauiContext mauiContext)
+		{
+			var platformContent = content.ToPlatform(mauiContext);
+
+			var defaultBrush = new UI.Xaml.Media.SolidColorBrush(Colors.Transparent.ToWindowsColor());
```

Comment:
Should we hardcode the Transparent here? I wonder if we should usa something like a Resource so users could override it if they want at platform level